.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,47 @@
+---
+name: Bug Report
+about: Create a bug report to help us improve our operating system.
+title: '[BUG] Brief description of the problem'
+labels: 'bug'
```

Comment:
To be consistent with other category labels, the `bug` label should be renamed to `C-bug`, right?

Reply:
Yes, it should be renamed.